mis_builder/tests/test_aep.py

```diff
@@ -24,7 +24,14 @@ def setUp(self):
         self.curr_year = datetime.date.today().year
         self.prev_year = self.curr_year - 1
         # create company
-        self.company = self.res_company.create({"name": "AEP Company"})
+        self.company = self.res_company.create(
+            {
+                "name": "AEP Company",
+                "country_id": self.env["res.country"]
+                .search([("code", "=", "US")], limit=1)
+                .id,
```

Comment:
```suggestion
                "country_id": (
                    self.env["res.country"]
                    .search([("code", "=", "US")], limit=1)
                    .id
                ),
```

A little detail: this is nicer to read.

Reply:
pre-commit changes it

Reply:
It looks better now, thanks.